ext/eventcart/CRM/Event/Cart/BAO/MerParticipant.php

```diff
@@ -9,6 +9,8 @@
  +--------------------------------------------------------------------+
  */
 
+use Civi\Api4\EventCartParticipant;
+
 /**
  * Class CRM_Event_Cart_BAO_MerParticipant
```

Comment:
Are MerParticipants... when mermaids register for your event?

Reply:
it's gender neutral